github_connector/models/github_analysis_rule_info.py

```diff
@@ -0,0 +1,38 @@
+# Copyright (C) 2020: Odoo Community Association (OCA)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import logging
+
+from odoo import api, fields, models
+
+_logger = logging.getLogger(__name__)
+
+
+class GithubAnalysisRuleInfoMixin(models.AbstractModel):
+    _name = "github.analysis.rule.info.mixin"
+    _description = "Github Analysis Rule Mixin"
+
```

Comment:
Maybe you can reuse the 'github.analysis.rule.mixin' here

Reply:
Interesting, it's possible to add and more re-use mixin function.